src/index.js

```diff
@@ -1,51 +1,60 @@
 const DEFAULT_ENVIRONMENTS = [
   {
-    displayName: 'local',
-    host: /(^localhost(:[0-9]+)?$|\.test$)/,
+    displayName: "local",
+    host: /(^localhost(:[0-9]+)?$|\.test$)/
   },
   {
-    displayName: 'dev',
+    displayName: "dev",
     host: /^([a-z0-9-]*-)?dev\./,
-    backgroundColor: '#fcd116',
+    backgroundColor: "#fcd116"
   },
   {
-    displayName: 'qa',
+    displayName: "qa",
     host: /^([a-z0-9-]*-)?qa\./,
-    backgroundColor: '#23b7fb',
-    foregroundColor: '#fff',
+    backgroundColor: "#23b7fb",
+    foregroundColor: "#fff"
   },
   {
-    displayName: 'preview',
+    displayName: "preview",
     host: /^([a-z0-9-]*-)?preview\./,
-    backgroundColor: '#22BC66',
-    foregroundColor: '#fff',
+    backgroundColor: "#22BC66",
+    foregroundColor: "#fff"
   }
-]
+];
 
 export default (environments = DEFAULT_ENVIRONMENTS) => {
   let environment = null;
+
   for (let i = 0; i < environments.length; i++) {
     if (environments[i].host.test(window.location.host)) {
-      environment = environments[i]
+      environment = environments[i];
       break;
     }
   }
 
   // Create the badge and add it to the page:
   if (environment) {
-    const { displayName, backgroundColor = '#ddd', foregroundColor = '#000' } = environment;
-    const environmentBadge = document.createElement('div');
-    environmentBadge.innerHTML = '\
-        <div style="display: block; position: absolute; top: -50px; left: -50px; width: 100px; height: 100px; background: ' + backgroundColor + '; color: ' + foregroundColor + '; transform: rotate(-45deg); z-index: 9999;">\
+    const {
+      displayName,
+      backgroundColor = "#000", //#cbd5e0
+      foregroundColor = "#fff"
+    } = environment;
+    const environmentBadge = document.createElement("div");
+    environmentBadge.innerHTML =
+      '\
```

Comment:
Can we use a template string so that this isn't quite so awkward to read after Prettier runs on it?

```js
environmentBadge.innerHTML = `
  <div style="display: block; position: fixed; top: -50px; left: -50px; width: 100px; height: 100px; background: ${backgroundColor}; color: ${foregroundColor}; transform: rotate(-45deg); z-index: 9999;">
    <span style="display: block; position: relative; top: 85%; font-size: 12px; text-align: center; font-weight: bold; text-transform: uppercase; transform: translateY(-50%); cursor: default;">
      ${displayName}
    </span>
  </div>
`;
```

Reply:
Yep, yep. Forgot we discussed switching it to a template string 👍